logging/dedicated-cluster-logging.adoc

```diff
@@ -3,6 +3,8 @@
 = Configuring OpenShift Logging in {product-title}
 include::modules/common-attributes.adoc[]
 
+toc::[]
```

Comment:
Thank you for adding this line. I checked in a local build and there is still no TOC, but that is now because there are no titles in the assembly. I realise this is out of scope for this PR, given that this assembly is not in the topic map and subsequently not included in the build.

Reply:
ok, thanks!